plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/ImplementStddevSamp.java

```diff
@@ -42,8 +42,7 @@
     public Pattern<AggregateFunction> getPattern()
     {
         return basicAggregation()
-                // TODO (https://github.com/trinodb/trino/issues/6189): remove stddev, an alias, from the list & simplify the pattern
-                .with(functionName().matching(name -> "stddev".equals(name) || "stddev_samp".equals(name)))
+                .with(functionName().matching(name -> "stddev".equals(name)))
```

Comment:
This could be simplified to `.with(functionName().equalTo("stddev"))`

Reply:
@kasiafi , I've addressed your comment in c12f028. Thanks for your review again.

Reply:
Thanks for updating! It looks good to me. @martint could you please take a look?